code/modules/transport/pods/vehicle.dm

```diff
@@ -7,6 +7,7 @@
 	anchored = ANCHORED
 	stops_space_move = 1
 	status = REQ_PHYSICAL_ACCESS
+	step_size = 16
```

Comment:
```suggestion
```
`step_size` is locked to multiples of `32` due to `world.movement_mode`, this line does nothing